.github/CODEOWNERS

```diff
@@ -1,20 +1,25 @@
 # See https://github.com/scottrigby/prometheus-helm-charts/issues/12
 # https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+# https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
-charts/prometheus/ @gianrubio @zanhsieh @Xtigyro
-charts/prometheus-adapter/ @mattiasgees @steven-sheehy @hectorj2f
-charts/prometheus-blackbox-exporter/ @desaintmartin @gianrubio
-charts/prometheus-cloudwatch-exporter/ @gianrubio @torstenwalter @asherf
-charts/prometheus-consul-exporter @timm088
-charts/prometheus-couchdb-exporter @gkarthiks
-charts/prometheus-mongodb-exporter @steven-sheehy
-charts/prometheus-mysql-exporter @juanchimienti @monotek
-charts/prometheus-nats-exporter @okgolove @caarlos0
-charts/prometheus-node-exporter @gianrubio @vsliouniaev
-charts/prometheus-operator @vsliouniaev @bismarck @gianrubio
-charts/prometheus-postgres-exporter @gianrubio
-charts/prometheus-pushgateway @gianrubio @cstaud
-charts/prometheus-rabbitmq-exporter @juanchimienti
-charts/prometheus-redis-exporter @acondrat
-charts/prometheus-snmp-exporter @miouge1
-charts/prometheus-to-sd @acondrat
+# The repo admins team will be the default owners for everything in the repo.
+# Unless a later match takes precedence, they will be requested for review when someone opens a pull request.
+* @prometheus-community/helm-charts-admins
+
+/charts/prometheus/ @gianrubio @zanhsieh @Xtigyro
```

Comment:
Nice catch on the preceding slash. As you said it's unlikely but possible for nested dirs in other charts to have one of these same names. Better to be explicit 👍